### ISSUE TITLE
Tests fail sporadically with error "IO Failed to bind to 0.0.0.0/0.0.0.0:XXXX"

### ISSUE THREAD

**Issue by marwyg** [opencast/opencast#5948]
### Describe the bug

Occasionally, an error occurs during the build process of Opencast when running tests. The error message can be found here: [GitHub Actions Log](https://github.com/opencast/opencast/actions/runs/9597736269/job/26467429541#step:7:7887).

### To Reproduce

1. Initiate the build process for Opencast. (Create a PR for example)
2. Observe the error (note: it does not always appear).

### Expected behavior

The build process and tests should complete successfully without errors.

### Stack trace
See the error message linked above for details.

### Additional context

The error is hard to reproduce as it seems to be caused by an already occupied server port, which is randomly assigned.

**Comment by marwyg** [comment]
Hopefully fixed by #5949

**Comment by lkiesow** [comment]
Since the GitHub Action logs are not preserved forever, here is the relevant error from the logs:

```

Error:  Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.052 s <<< FAILURE! -- in org.opencastproject.adminui.endpoint.AclEndpointTest
Error:  org.opencastproject.adminui.endpoint.AclEndpointTest -- Time elapsed: 0.049 s <<< ERROR!
java.io.IOException: Failed to bind to 0.0.0.0/0.0.0.0:58344
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349)
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310)
	at org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80)
	at org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
	at org.eclipse.jetty.server.Server.doStart(Server.java:401)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
	at org.opencastproject.test.rest.RestServiceTestEnv.setUpServer(RestServiceTestEnv.java:130)
	at org.opencastproject.adminui.endpoint.AclEndpointTest.oneTimeSetUp(AclEndpointTest.java:58)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:316)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:240)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:214)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
Caused by: java.net.BindException: Address already in use
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:459)
	at java.base/sun.nio.ch.Net.bind(Net.java:448)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227)
	at java.base/sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80)
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344)
	... 28 more

[…]

[INFO] Results:
[INFO] 
Error:  Errors: 
Error:    AclEndpointTest.oneTimeSetUp:58 » IO Failed to bind to 0.0.0.0/0.0.0.0:58344
```
